src/sentry/event_manager.py

```diff
@@ -284,26 +284,37 @@ def get_stored_crashreports(cache_key: str | None, event: Event, max_crashreport
 
 
 class ScoreClause(Func):
-    def __init__(self, group=None, last_seen=None, times_seen=None, *args, **kwargs):
+    def __init__(
+        self,
+        group: Group | None = None,
+        last_seen: datetime | None = None,
+        times_seen: int | None = None,
+        *args: Any,
+        **kwargs: Any,
+    ):
         self.group = group
         self.last_seen = last_seen
         self.times_seen = times_seen
         # times_seen is likely an F-object that needs the value extracted
         if hasattr(self.times_seen, "rhs"):
-            self.times_seen = self.times_seen.rhs.value
+            self.times_seen = self.times_seen.rhs.value  # type: ignore[union-attr]
```

Comment:
we should not be ignoring the type error -- this should include a check that it is the correct type